adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMRDDFunctions.scala

```diff
@@ -81,7 +131,25 @@ class ADAMRDDFunctions[T <% IndexedRecord: Manifest](rdd: RDD[T]) extends Serial
       ContextUtil.getConfiguration(job)
     )
   }
+}
+
+@deprecated("Extend ADAMRDDFunctions and mix in GenomicRDD wherever possible in new development.",
+  since = "0.20.0")
+class ConcreteADAMRDDFunctions[T <% IndexedRecord: Manifest] private[rdd] (val rdd: RDD[T]) extends ADAMRDDFunctions[T] {
```

Comment:
what is this used for?  if just for the implicit conversion could it be private to `ADAMContext`?